src/os/bluestore/BlueStore.h

```diff
@@ -328,25 +340,25 @@ class BlueStore : public ObjectStore,
       return nullptr;
     }
 
-    Blob *new_blob(Cache *c) {
+    BlobRef new_blob(Cache *c) {
       int64_t id = get_new_id();
       Blob *b = new Blob(id, c);
+      b->get();
```

Comment:
If BlobRef is an intrusive_ptr there shouldn't be any explicit get/put calls anywhere (except the intrustive ptr hooks)...